2022/Q4/artifacts/PoS-pseudocode/PoS-model.md

```diff
@@ -272,14 +277,23 @@ func unbond(validator_address, delegator_address, unbond_amount)
           remain = 0
```

Comment:
Should `remain = 0` come after the loop over the slashes and `validators[validator_address].total_unbonds[cur_epoch+pipeline_length] = {UnbondRecord{amount: remain, start: start}}`?

Reply:
Thanks @brentstone! You are completely right. Fixed.